### DISCUSSION TITLE
Prevent users from posting in/seeing structures they haven't created

### DISCUSSION THREAD

**Discussion by NicolasBarse** [craftcms/cms#9037] [Ideas] [answered]
### Description

I'm building a website with franchises where users can create their own franchise page and post articles in their page. It's a two entry-type structure, one entry for the franchise page and another one for the articles.

Currently craft allows me to prevent them editing entries they haven't created themselves. However when creating an article and selecting the parent, they can select any other franchise which is quite of an issue.

It would be great to have a way to hide or **prevent users from posting anything in a parent that they haven't created or prevent them from seeing those parents if they haven't created them.**

**Comment by brandonkelly** [comment, marked as answer]
Related: #6280
